forch/authenticator.py

```diff
@@ -33,7 +33,12 @@ def __init__(self, auth_config, auth_callback=None, radius_query_object=None):
         radius_info = auth_config.radius_info
         radius_ip = radius_info.server_ip
         radius_port = radius_info.server_port
-        secret = radius_info.secret
+        secret = None
```

Comment:
better to use an if-else chain, rather than assigning multiple times. It's functionally the same, but is more "clean" in terms of understanding and refactoring. E.g.,:

if A:
  secret = f(A)
elif B:
  secrect = foo
else
  secret = None

Or consider extracting to a function, secret = self._get_radius_secret()